src/api/backend.ts

```diff
@@ -0,0 +1,20 @@
+import { ladeTermine, clearStore } from '../actions'
+
+const API_URL = 'http://localhost:8080/api/termine';
+
+export const loadTermine = () => (dispatch: any) => {
+  dispatch(clearStore());
```

Comment:
Falls es einfach als Übung gedacht ist, dann ist es ok. Aber ansonsten würde ich nicht einfach den Store beim termine Laden löschen oder wenn du dies wirklich so machen willst müsste die Funktion loadTermineFromBackend heissen.
Weiter müssten wir als default dann die Termine aus dem Local Store zuerst laden und diese danach mit dem Backend ergänzen, wie wir es in der Spezifikation erwähnt haben.

Reply:
ok, das müssten wir genauer anschauen. Der Grund warum ich den Store gelöscht habe war, dass der Fetch zweimal ausgeführt wurde und die Termine doppelt im Store waren.